ibis/pandas/tests/test_operations.py

```diff
@@ -926,16 +926,14 @@ def test_cast_to_decimal(t, df, type):
          lambda x: x.clip(lower=0, upper=1)),
         (lambda x: x.clip(lower=0, upper=1.0),
          lambda x: x.clip(lower=0, upper=1.0)),
+        (lambda x: x.quantile([0.25, 0.75]),
+         lambda x: x.quantile([0.25, 0.75])),
     ]
 )
 def test_arraylike_functions_transforms(t, df, ibis_func, pandas_func):
-    if isinstance(pandas_func, Exception):
-        with pytest.raises(pandas_func):
-            ibis_func(t.float64_with_zeros).execute()
-    else:
-        result = ibis_func(t.float64_with_zeros).execute()
-        expected = pandas_func(df.float64_with_zeros)
-        tm.assert_series_equal(result, expected)
+    result = ibis_func(t.float64_with_zeros).execute()
+    expected = pandas_func(df.float64_with_zeros)
+    tm.assert_series_equal(result, expected)
```

Comment:
We'll return a `list` here since this is a reduction to a single array-scalar so `assert` is probably good enough.